scripts/deploy.js

```diff
@@ -124,6 +124,21 @@ async function httpsGetWithRedirects(url) {
   });
 }
 
+// Function generates a warning or error depending on the contract bytecode size
+// It is only approximation based on the bytecode (constructor + bytecode)
+// Run forge build --sizes for stats about deployed bytecode size
+// Max contract size is 24kb
+// We set a warning threshold at 18kb
+function checkContractSize(spinner, bytecode) {
+  let warning_threshold = 18000;
+  let error_threshold = 24000;
```

Comment:
Put these as well-named constants at the top of the file, please. `CONTRACT_SIZE_WARNING_THRESHOLD_BYTES` and `CONTRACT_SIZE_ERROR_THRESHOLD_BYTES`.